server/database/migrations/20200523192707-create-prospect.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Prospects', {
```

Comment:
Isn't dev already have Prospects table or model?

Reply:
But I built this branch off of yours before the dev had Prospects model so I would need to resolve conflicts once the changes approved